WebAPI/Entity/NewsBlogs.cs

```diff
@@ -16,7 +16,15 @@ public class NewsBlogs : IEntity<int>
 
         public string Category { get; set; }
 
-        [MaxLength(50)]
+		public string NewsTitle { get; set; }
```

Comment:
alignment for all

Reply:
but in my local it is properly aligned
I dont know here it is behaving different manner